pkg/deploy/api/v1beta3/types.go

```diff
@@ -42,6 +44,8 @@ const (
 	DeploymentStrategyTypeRecreate DeploymentStrategyType = "Recreate"
 	// DeploymentStrategyTypeCustom is a user defined strategy.
 	DeploymentStrategyTypeCustom DeploymentStrategyType = "Custom"
+	// DeploymentStrategyTypeRolling uses the Kubernetes RollingUpdater.
+	DeploymentStrategyTypeRolling DeploymentStrategyType = "Rolling"
```

Comment:
Default to rolling (add a defaulting func in v1beta3, don't default in validations anymore)